ci/cloudbuild/build.sh

```diff
@@ -168,6 +172,7 @@ if [[ "${DOCKER_BUILD}" = "true" ]]; then
     "--rm"
     "--user=$(id -u):$(id -g)"
     "--env=USER=$(id -un)"
+    "--env=TZ=:UTC"
```

Comment:
I'm having second thoughts about the precise incantation.  `TZ=:UTC` will probably only work on tzdb-based implementations that compile the "backward" rules, and I'm not sure who does that.

We're probably better off with `TZ=UTC0`, which will work on any system that supports the POSIX TZ spec, and that includes any Unix-like libc, plus Windows I believe.  Abseil/CCTZ will actually fail to load "UTC0" (unlike bare "UTC", which we special case), but it will then fall back to UTC, so that's OK.  [But I think I'll tweak CCTZ so that loading "UTC0" returns success too.]

Reply:
Done.